### ISSUE TITLE
Can't enable app in Nextcloud 22.

### ISSUE THREAD

**Issue by severinkaderli** [mtierltd/timetracker#124]
When I try to enable the app (Version 0.0.59) in Nextcloud 22 I get the following error:

```
The appinfo/database.xml file is not longer supported. Used in timetracker
```

**Comment by elpraga** [comment]
Is there a fix coming up?

**Comment by reinerj** [comment]
yes

**Comment by HieronymusCH** [comment]
Can we be of any assistance?

**Comment by mtierltd** [comment]
Due to some not great design decisions in nextcloud, this is taking longer now as the database table
names cannot be longer than 27 chars in migrations and because of that we need to rename our tables
and make sure that your old tables are migrated over properly to the new one.

**Comment by mtierltd** [comment]
You can try and test commit 2917a4dc5b6be91d74d5ecea323e07b27a6f2e93 but make sure to have a database backup.
You will probably have to run
`occ migrations:execute timetracker 000000Date20210719124731` manually, thank you nextcloud!

**Comment by mtierltd** [comment]
fixed by >0.0.60
